aiokafka/conn.py

```diff
@@ -414,7 +414,7 @@ def port(self):
     def send(self, request, expect_response=True):
         if self._writer is None:
             raise Errors.KafkaConnectionError(
-                "No connection to broker at {0}:{1}"
+                "No connection to broker at {}:{}"
```

Comment:
`{self._host}:{self._port}`? And in other similar places below

Reply:
Sure thing. Unfortunately, **pyupgrade** cannot `pyupgrade` multiline strings, so I had to do it half-manually with a regex.

FYI, related [issue](https://github.com/asottile/pyupgrade/issues/55) in the **pyupgrade** repo